src/app/app.component.ts

```diff
@@ -1,16 +1,19 @@
 import { Component } from '@angular/core';
 import { FormBuilder } from '@angular/forms';
+import {MatStepper} from '@angular/material/stepper';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss'],
 })
 export class AppComponent {
+  step = 1;
+  isFingeringDeviceConnected = false;
 
-  constructor(private formBuilder: FormBuilder) {}
+  constructor(private formBuilder: FormBuilder) {
```

Comment:
it's not your change. But please take a chance to remove unused stuffs

Reply:
@hoangngk Removed